pr-request-template.md

```diff
@@ -0,0 +1,77 @@
+# Pull Request Template
+
+Thank you for your contribution to Alumnium!
+Please fill out this template completely to help us understand and review your PR efficiently.
```

Comment:
WDYT if we relax a bit on the rules and not force contributors to fill in everything? Some of the sections, like Related issue, won't be relevant in all cases. Other idea is to cut down the sections a bit.
```suggestion
Please fill out this template to help us understand and review your PR efficiently.
```